nncf/torch/dynamic_graph/wrappers.py

```diff
@@ -87,65 +89,27 @@ def wrapped(*args, **kwargs):
         ctx.in_operator = True
 
         try:
-            op_name = operator_info.name
-            op_address = ctx.get_caller_context(op_name)
-            str_op_address = str(op_address)
             if operator_info.skip_trace:
                 result = operator(*args, **kwargs)
             elif ctx.is_forwarding:
                 from nncf.torch.dynamic_graph.trace_functions import forward_trace_only
                 result = forward_trace_only(operator, *args, **kwargs)
             else:
-                node = None
                 op_name = operator_info.name
                 op_address = ctx.get_caller_context(op_name)
-
-                layer_attrs = None
-                ignored_algos = []
-
-                # Collect module attributes, if required
-                if ctx.trace_dynamic_graph and not ctx.in_skipped_block:
-                    if op_name in OP_NAMES_REQUIRING_MODULE_ATTRS:
-                        curr_module = ctx.get_current_module()
-                        if curr_module is None:
-                            raise RuntimeError("Operation {} requires module attributes, "
-                                               "but it was executed outside any module".format(op_name))
-                        layer_attrs = _get_layer_attributes(curr_module, op_name)
-                        if isinstance(curr_module, _NNCFModuleMixin):
-                            ignored_algos = deepcopy(curr_module.ignored_algorithms)
-
                 ctx.register_operator_call(op_address.operator_name, op_address.scope_in_model)
-                op_input = OperatorInput(list(args), kwargs)
-                processed_input = ctx.execute_pre_hooks(op_address, op_input)
-
-                if ctx.trace_dynamic_graph and not ctx.in_skipped_block:
-                    tensor_metas = make_tensor_metas(processed_input)
-                    node = ctx.find_operator_node(tensor_metas, op_address)
 
-                args = tuple(processed_input.op_args)
-                kwargs = processed_input.op_kwargs
-
-                if not ctx._elastic_depth or not ctx.in_skipped_block:
-                    result = operator(*args, **kwargs)
-                else:
+                if ctx.elastic_depth and ctx.in_skipped_block:
```

Comment:
I've significantly refactored `wrapped_operator` function.
@asenina @vshampor  please pay extra attention to it

Reply:
now it's not calling pre/post ops in the skipping mode.
the logic is the same, reorganized code to reduce number of if-conditions

Reply:
LGTM